core/src/main/java/org/frankframework/util/MessageUtils.java

```diff
@@ -95,6 +100,32 @@ public static MessageContext getContext(Iterator<MimeHeader> mimeHeaders) {
 		return result;
 	}
 
+	public static MessageContext getContext(HttpResponse httpResponse) {
+		MessageContext result = new MessageContext();
+		HttpEntity entity = httpResponse.getEntity();
+		if(entity != null) {
+			result.withSize(entity.getContentLength());
+			Header contentType = entity.getContentType();
+			if(contentType != null) {
+				result.withMimeType(contentType.getValue());
+			}
+		} else {
+			Header contentTypeHeader = httpResponse.getFirstHeader(HttpHeaders.CONTENT_TYPE);
```

Comment:
Oh die kan je ook mooi op regel 94 gebruiken denk ik

Reply:
Die als in de constant bedoel je?